### PR TITLE
Fixed wet/dry override & code clean-up

### PR DESCRIPTION
**Pull request by ronvandegraaf**
- Fixed wet/dry function
- Changed labels open/close to wet/dry
- Small code clean-up, removed tabs
- Tested device handler

### CODE REVIEW

**Comment by gn0st1c** [comment]
this is great. i, too, don't like trailing spaces, not aligned stuff and so on.
this code now looks pretty much the same as mine :)

except for the tabs vs spaces, i'm pro tabs.
https://www.youtube.com/watch?v=SsoOG6ZeyUI

**Comment by ronvandegraaf** [comment]
Added extra commit to this pull request: Changed the colors for dry (=blue) and wet (=orange), like the door/window sensors. To give more attention in your 'Things' list.

File: devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy [hunk 1]
```diff
@@ -47,63 +47,63 @@ metadata {
...
         }
         valueTile("lastcheckin", "device.lastCheckin", decoration: "flat", inactiveLabel: false, width: 4, height: 1) {
             state "default", label:'Last Checkin:\n${currentValue}'
         }
         valueTile("lastopened", "device.lastOpened", decoration: "flat", inactiveLabel: false, width: 4, height: 1) {
             state "default", label:'Last Wet:\n${currentValue}'
         }
-        standardTile("resetClosed", "device.resetClosed", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
-            state "default", action:"resetClosed", label: "Override Close", icon:"st.contact.contact.closed"
+        standardTile("resetWet", "device.resetCWet", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
```
Comment (veeceeoh) [inline devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy]: "device.resetCWet" should be "device.resetWet"

File: devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy [hunk 2]
```diff
@@ -47,63 +47,63 @@ metadata {
         attribute "lastOpened", "String"
```
Comment (veeceeoh) [inline devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy]: This attribute should be changed to "lastWet"
Sorry - I also missed this in yesterday's push!

File: devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy [hunk 3]
```diff
@@ -47,63 +47,63 @@ metadata {
...
         attribute "lastOpenedDate", "Date"
```
Comment (veeceeoh) [inline devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy]: This attribute should be changed to "lastWetDate"

**Comment by veeceeoh** [comment]
Apologies! I see that I missed fixing some of the open / close references in my push yesterday. Looks like you caught nearly all of it now - there's just a couple of attributes defined in the metadata section that also need to be changed.

**Comment by ronvandegraaf** [comment]
Changed it in my last commit. We maken some great progress on these device handlers now.

**Comment by veeceeoh** [comment]
ranvandegraff - do you have problems with the appearance of the Battery Change tile? It looks incorrect in the ST App on my iPhone. I will post screenshots on Brian's ST forum thread.

**Comment by ronvandegraaf** [comment]
Yes, same problem for me (iPhone Plus).

**Pull request merged**